src/utils.js

```diff
@@ -28,6 +28,10 @@ export function getDisplayName(WrappedComponent: string | Component): string {
 }
 
 export function validateSpacingProps(props: SpacingProps) {
+  if (isProd) {
```

Comment:
any idea if this is enough for uglify to dead code eliminate the rest of this function?